wagtailvideos/models.py

```diff
@@ -199,7 +204,7 @@ def video_tag(self, attrs=None):
 
         mime = mimetypes.MimeTypes()
```

Comment:
can delete this line, it's not `mime` isn't used after here

Reply:
You are totally right! Fixed :)